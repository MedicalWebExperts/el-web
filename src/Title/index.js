import React from 'react';
import PropTypes from 'prop-types';

const propTypes = {
  text: PropTypes.string.isRequired,
};

const Title = ({ text }) => (<h1>{text}</h1>);

Title.propTypes = propTypes;
export default Title;
