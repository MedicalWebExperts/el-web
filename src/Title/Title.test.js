import React from 'react';
import renderer from 'react-test-renderer';

import Title from './index';

describe('Snapshots', () => {
  it('should match snapshot of Title', () => {
    const component = renderer.create(<Title text="Hello World" />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
