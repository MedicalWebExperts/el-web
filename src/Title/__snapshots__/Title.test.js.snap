// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshots should match snapshot of Title 1`] = `
<h1>
  Hello World
</h1>
`;
